.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Fantomas 6.0.0-alpha-010
+c9d0d84ff5b82641deb1a7d315dfa92307b7997b
```

Comment:
By the way, since this repository requires squash-merging, the only way to do this is to create another PR once this one goes in, listing the hash of the squash commit.

Reply:
Ah yeah, thanks for the heads-up. Will do.

Reply:
Could you please revert the change to this file, given that it won't do anything? (Assuming you're actually proposing this PR and it isn't just to see how Fantomas v6 behaves.)

Reply:
done